docs: setup-a-validator.md  Add cluster specific note (#3064)
anza-xyz/agave: docs/src/operations/setup-a-validator.md

doing in this script. Also refer to the section on
[best practices for operating a validator](./best-practices/general.md).

> This startup script is specifically intended for testnet. Other startup
> script examples are contained in 
> [the clusters section.](./../clusters/available.md).
> 

Reviewer: micro-nit:

```suggestion
This startup script is specifically intended for testnet. For more startup script examples intended for other clusters, refer to the
[clusters section.](./../clusters/available.md).
```
